ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -166,7 +167,20 @@ def main(argv=sys.argv[1:]):
     if jobs:
         cmd.extend(['-j', '%d' % jobs])
     if args.enable_misra_checks:
-        cmd.extend(['--addon=misra'])
+        # The MISRA rule text is copyrighted. However, if the user happens to have a misra.json
+        # configuration file in the $HOME directory, use it.
+        #
+        # Here's an example misra.json:
+        # {
+        #   "script": "/usr/lib/x86_64-linux-gnu/cppcheck/addons/misra.py",
+        #   "args": [
+        #      "--rule-texts=/home/spaceros-user/misra-rules.txt",
+        #      "--suppress-rules 17.3,21.12"
+        #   ]
+        # }
+
+        misra_json = os.path.join(os.environ['HOME'], 'misra.json')
```

Comment:
```suggestion
        misra_json = os.path.join(os.path.expanduser('~'), 'misra.json')
```

Thanks for iterating. Using `os.path.expanduser` will work on Windows as well and fall back to looking up the current user in `/etc/passwd` if `HOME` is unset for some reason. https://docs.python.org/3/library/os.path.html#os.path.expanduser

Accepting this suggestion will also require adding `import os.path` to the imports at the top of the file.